packages/storybook/src/stories/playground/customComponents.tsx

```diff
@@ -42,7 +41,7 @@ export function NoRowsComponent() {
   );
 }
 
-export function PaginationComponent(props: GridBaseComponentProps & { color?: 'primary' }) {
+export function PaginationComponent(props) {
```

Comment:
why remove the types? How do you achieve what the story is doing?

Reply:
I should have probably fixed the type because it is no longer the one that was set. It accepts additional stuff from the HTML attributes. I removed it because it was breaking the linter.

Reply:
Could you put them back please

Reply:
I need to fix it, yes.

Reply:
reverted